components/script/dom/stylesheet.rs

```diff
@@ -44,6 +46,11 @@ impl StyleSheetMethods for StyleSheet {
         self.href.clone()
     }
 
+    // https://drafts.csswg.org/cssom/#dom-stylesheet-ownernode
+    fn GetOwnerNode(&self) -> Option<DomRoot<Element>> {
+        self.downcast::<CSSStyleSheet>().unwrap().get_owner()
```

Comment:
Might as well use `and_then` instead of unwrapping here.

Reply:
Fixed